core-projects/xarray.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "xarray"
+avatar: https://numfocus.org/wp-content/uploads/2018/09/xarray-logo-square.png
+homepage: https://xarray.dev/
+repository: https://github.com/pydata/xarray
+pypi: https://pypi.org/project/xarray
+libraries-io: https://libraries.io/pypi/xarray
+license: https://github.com/pydata/xarray/blob/main/LICENSE
+license-type: Apache 2.0
+contact: shoyer
```

Comment:
We are searching for an alternate representative as @shoyer is quite overloaded with responsibilities. We will make a PR here with an update once we have our volunteer confirmed. For the interim, fine with leaving him on.